package.json

```diff
@@ -165,7 +165,7 @@
 		"reakit": "1.3.0",
 		"trim-html": "0.1.9",
 		"use-debounce": "3.4.3",
-		"wordpress-components": "npm:@wordpress/components@8.5.0",
+		"wordpress-components": "npm:@wordpress/components@11.1.1",
```

Comment:
I'm curious why we have this dep and the other one in `devDependencies`. What does this mean? Please excuse my ignorance - feel free to point me to docs 😄

Reply:
Sure thing! Form the [docs](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file):
> * "dependencies": Packages required by your application in production.
> * "devDependencies": Packages that are only needed for local development and testing.

In our case:
- `wordpress-components` is bundled in frontend scripts, so it's a dependency of the production builds. Because of that, it's listed in `dependencies`.
- `@wordpress/components` is extracted by `DependencyExtractionWebpackPlugin`, so it's not bundled in the production builds, so it's listed in `devDependencies`.

For reference, dependencies were re-ordered in this #1663, and there was a similar discussion about `@wordpress/components`: https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1633#discussion_r370339806.